cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -315,7 +333,8 @@ var createModeSubcommands = map[string]modeSubcommand{
 		Setup: func(clientGetter util.ClientGetter, subcmd *cobra.Command, o *CreateOptions) {
 			subcmd.Use += " [--replicas [WORKER_GROUP]=REPLICAS]" +
 				" [--min-replicas [WORKER_GROUP]=MIN_REPLICAS]" +
-				" [--max-replicas [WORKER_GROUP]=MAX_REPLICAS]"
+				" [--min-replicas [WORKER_GROUP]=MIN_REPLICAS]" +
```

Comment:
same as previous issue, max replicas is gone and min replicas is duplicated

Reply:
Thanks!